client/src/EventDetail.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { PollingEvent } from "./types.ts";
```

Comment:
tady, v EventsList i App.tsx mas spatne importy, eslint/typescript si stezuje. Melo by to byt
```
import { type PollingEvent } from "./types.ts";
```